From ff8cfbe1e992cfd40c01541324c6f0155da26775 Mon Sep 17 00:00:00 2001
From: Jason Cawood <jasoncawood@gmail.com>
Date: Thu, 24 Oct 2019 21:08:44 -0700
Subject: [PATCH 10/25] fixed the duplication typo.

I meant to update the file to include the new product id, but hastily included the id 0120 twice.  This corrects that error and includes 0121

Signed-off-by: Gustavo B. Schenkel <gustavo.schenkel@gmail.com>
---
 u2f.conf.sample | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/u2f.conf.sample b/u2f.conf.sample
index 040b5c2..6d89540 100644
--- a/u2f.conf.sample
+++ b/u2f.conf.sample
@@ -6,7 +6,7 @@ notify 100 {
 	match "subsystem"	"DEVICE";
 	match "type"		"ATTACH";
 	match "vendor"		"0x1050";
-	match "product"		"(0x0113|0x0114|0x0115|0x0116|0x0120|0x0120|0x0200|0x0402|0x0403|0x0406|0x0407|0x0410)";
+	match "product"		"(0x0113|0x0114|0x0115|0x0116|0x0120|0x0121|0x0200|0x0402|0x0403|0x0406|0x0407|0x0410)";
 	action	"chgrp u2f /dev/$cdev; chmod g+rw /dev/$cdev";
 };
 
-- 
2.32.0

